test/metabase/lib/field_test.cljc

```diff
@@ -684,7 +739,7 @@
                :semantic-type :type/Category,
                :is-from-join false,
                :long-display-name "Price: Auto binned",
-               :display-name "Price",
+               :display-name "Price: Auto binned",
```

Comment:
I think this is the proper display name for a binned column, so I consider this is a bug fix.